FrontEnd/src/assets/scss/utils/_global-vars.scss

```diff
@@ -47,6 +47,7 @@ $black-color: #202020;
 $link-color: #6d83e7;
 $background-color: #fffaf8;
 $border-color: #f8e6d8;
+$background-blur-dark:rgba(32, 32, 32, 0.6);
```

Comment:
Nitpick: space between colon and rgba. Prettier isnt working if this is happening.